### PR TITLE
Adding kp-mariappan-ramasamy and xv-ian-c to code owners

### PR DESCRIPTION
**Pull request by expressvpn-tom-l**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (expressvpn-andre-l) [inline .github/CODEOWNERS:2]: I believe Maari would like to use @kp-mariappan-ramasamy now, see https://github.com/expressvpn/wolfssl-rs/blob/main/.github/CODEOWNERS

Comment (expressvpn-tom-l) [reply .github/CODEOWNERS]: Done.

Comment (kp-mariappan-ramasamy) [reply .github/CODEOWNERS]: Thanks @expressvpn-andre-l @expressvpn-tom-l

**Comment by kp-mariappan-ramasamy** [comment]
@expressvpn-tom-l @expressvpn-andre-l 
At present, both myself and Ian had no push permission to this repo. 
And Github is throwing errors in CODEOWNERS file

I have raised a PR in github-iac to add push permission. We can merge this after merging iac pr.

**Comment by kp-mariappan-ramasamy** [comment]
Thanks @expressvpn-andre-l for helping merging iac pr
I have accepted the invitation. Once Ian accepted, we are good to merge

**Review by expressvpn-andre-l — APPROVED** [review]

**Pull request merged**
